### DISCUSSION TITLE
[ Feature Request ] Allow change file attributes in all subfolders

### DISCUSSION THREAD

**Discussion by Augusto7743** [doublecmd/doublecmd#1149] [Suggestions]
Selecting a folder properties to be changed (read only or etc) not is possible run the "set properties" in all files in sub folders.
Unhappily the user need to do the same command several times in all sub folders.

**Comment by alexx2000** [comment]
Menu->"Files"->"Change attributes..."->"Including subfolders"

**Comment by Augusto7743** [comment]
Thanks for reply.
Well ... I say about change attributes properties when using the file panel , but if selecting a folder and using file menu works is possible run in sub folders.
Directly running attributes if selecting a folder in file tab is good.

Have a nice week.
